modules/em/test/test_mrc_indexing.py

```diff
@@ -0,0 +1,57 @@
+from __future__ import print_function
+import IMP
+import IMP.test
+import IMP.em
+import os
+
+
+class Tests(IMP.test.TestCase):
+    """Test that MRC files with non-standard indexing are transposed
+       correctly"""
+
+    def test_centroid_similarity(self):
+        """The centroids of experimental and model-derived map should
+           be similar"""
+        in_mrc = self.get_input_file_name("emd-20362.mrc")
+        in_pdb = self.get_input_file_name("emd-20362.pdb")
+
+        mrw = IMP.em.MRCReaderWriter()
+        self.dmap = IMP.em.read_map(in_mrc, mrw)
+        header = self.dmap.get_header()
+        # Map doesn't set origin correctly; use n*start values instead
+        self.dmap.set_origin(
```

Comment:
This origin shift seems to be required to align the EM map.  Should this be done in MRCReaderWriter?

Reply:
Yes, Ilan and I talked about this already in regards to his issue #1054. I have a patch already to correct the origin for files like these. I'll apply that separately though since that's a different issue.

Reply:
Should be in the most recent round of commits. Please review and merge when you get a chance.